docs/src/content/docs/es/getting-started.mdx

```diff
@@ -93,37 +140,34 @@ Starlight es una integración Astro y se actualiza de la misma manera que cualqu
 <TabItem label="npm">
 
 ```sh
-# actualizar Starlight con npm
 npm install @astrojs/starlight@latest
 ```
 
 </TabItem>
 <TabItem label="pnpm">
 
 ```sh
-# actualizar Starlight con pnpm
 pnpm upgrade @astrojs/starlight --latest
 ```
 
 </TabItem>
 <TabItem label="Yarn">
 
 ```sh
-# actualizar Starlight con yarn
 yarn upgrade @astrojs/starlight --latest
 ```
 
 </TabItem>
 </Tabs>
 
-Puedes ver una lista completa de los cambios realizados en cada versión en [el registro de cambios de Starlight](https://github.com/withastro/starlight/blob/main/packages/starlight/CHANGELOG.md).
+Consulta el [registro de cambios de Starlight](https://github.com/withastro/starlight/blob/main/packages/starlight/CHANGELOG.md) para obtener una lista completa de los cambios realizados en cada versión.
 
 ## Solución de problemas de Starlight
 
-Tanto la [configuración del proyecto](/es/reference/configuration/) como la [configuración individual de frontmatter](/es/reference/frontmatter/) de cada página están disponibles en la sección de Referencia de este sitio. Utiliza estas páginas para asegurarte de que tu sitio Starlight está configurado y funcionando correctamente.
-
-Revisa la creciente lista de guías en la barra lateral para obtener ayuda para agregar contenido y personalizar tu sitio Starlight.
+Utiliza las páginas de referencia de [configuración del proyecto](/reference/configuration/) y [configuración del frontmatter de páginas individuales](/reference/frontmatter/) para asegurarte de que tu sitio de Starlight esté configurado y funcione correctamente. 
+Consulta las guías en la barra lateral para obtener ayuda en la adición de contenido y personalización de tu sitio de Starlight.
 
-Si no encuentras la respuesta en la documentación, visita la [documentación completa de Astro](https://docs.astro.build) para obtener la documentación completa de Astro. Es posible que tu pregunta se responda comprendiendo cómo funciona Astro en general, debajo de este tema Starlight.
+Si no encuentras la respuesta en la documentación, visita la [documentación completa de Astro](https://docs.astro.build) para obtener la documentación completa de Astro.
```

Comment:
And it probably also makes sense here to link to the Spanish version of the Astro Docs:

```suggestion
Si no encuentras la respuesta en la documentación, visita la [documentación completa de Astro](https://docs.astro.build/es/) para obtener la documentación completa de Astro.
```